src/cron/janitor.ts

```diff
@@ -3,6 +3,32 @@ import * as fs from 'fs';
 import { DB_FILE_STATUS, JANITOR_COPIES } from '../constants';
 import { DatabaseService } from '../services/database.service';
 
+export async function syncLogsDb(db: DatabaseService) {
+  console.log('janitor: sync logs with DB records');
+  const result = await db.query<{
+    id: number;
+    basename: string;
+    path: string;
+  }>(
+    `
+    SELECT id, basename, path
+    FROM logs
+    WHERE status = ?
+    ORDER BY id DESC
+    `,
+    [DB_FILE_STATUS.CopiedToObjectStore],
```

Comment:
Also applies to Rotated and Compressed. Could be a parameter for operations that rely on those files being there to call.